pontoon/base/views.py

```diff
@@ -384,7 +383,7 @@ def get_translation_history(request):
             "uid": "" if u is None else u.id,
             "username": "" if u is None else u.username,
             "date": t.date.strftime('%b %d, %Y %H:%M'),
-            "date_iso": t.date.isoformat() + offset,
+            "date_iso": t.date.isoformat(),
```

Comment:
This part of the changeset affects Translate.Current as well. I haven't tested the code yet, but I wonder if we need to make changes to `translate.js` as well.

Reply:
The only place I see a usage in `translate.js` is to fill [`<time datetime="...">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time#Valid_datetime_Values), which is never displayed to the user.

Reply:
Btw. latest activity tag on dashboards uses just `isoformat()`: `<time datetime="{{ latest_activity.date.isoformat() }}" ...>`.